endpoints/openrtb2/auction.go

```diff
@@ -1127,10 +1127,6 @@ func (deps *endpointDeps) validateUser(user *openrtb2.User, aliases map[string]s
 		// Creating ExtUser object to check if DigiTrust is valid
```

Comment:
Please reword this comment to not include `Digitrust` anymore

Reply:
Reworded!